docs/3.0.0-beta.x/concepts/models.md

```diff
@@ -91,12 +95,16 @@ If you are just starting out it is very convenient to generate some models with
 
 ### For Content Types models
 
-Use the CLI, and run the following command `strapi generate:model restaurant name:string description:text`.<br>Read the [CLI documentation](../cli/CLI.md) for more information.
+Use the CLI, and run the following command:
 
-This will create two files located at `./api/restaurant/models`:
+`strapi generate:model restaurant name:string description:text`
```

Comment:
Can you please use the tab system to have the `yarn`, `npx` and `strapi` commands

Like here - http://localhost:8080/documentation/3.0.0-beta.x/installation/cli.html#step-2-create-a-new-project

Reply:
done